kcbq-connector/src/main/java/com/wepay/kafka/connect/bigquery/write/row/BigQueryWriter.java

```diff
@@ -169,13 +170,20 @@ public void writeRows(PartitionedTableId table,
              todo possibly this page is inaccurate for bigquery, but the message we are getting
              suggest it's an internal backend error and we should retry, so lets take that at face
              value. */
+          logger.warn("BQ backend error: {}", err.code());
```

Comment:
Can you add a note to make it clear that we're going to retry? I'm afraid this error msg might freak me out otherwise.